app/build.gradle

```diff
@@ -101,4 +101,4 @@ dependencies {
     androidTestImplementation "androidx.compose.ui:ui-test-junit4:$compose_version"
     debugImplementation "androidx.compose.ui:ui-tooling:$compose_version"
     debugImplementation "androidx.compose.ui:ui-test-manifest:$compose_version"
-}
\ No newline at end of file
+}
```

Comment:
I think we should move these newline changes to #2, they aren't exactly relevant to this PR 🤔 What do you think? 😄